lib/model/message_list.dart

```diff
@@ -86,6 +87,30 @@ class MessageListView extends ChangeNotifier {
     notifyListeners();
   }
 
+  Message applyChangesToMessage(UpdateMessageEvent event, Message oldMessage) {
+    Map<String, dynamic> oldMessageJson = oldMessage.toJson();
+
+    //TODO: What other fields should be edited by the update event?
+    oldMessageJson["content"] = event.renderedContent;
+    oldMessageJson["lastEditTimestamp"] = event.editTimestamp;
```

Comment:
I noticed this edit timestamp wasn't displayed in the UI anywhere. Should that be part of this change?

Reply:
Noted! OK, resolving since this is out of scope.